src/components/editMeasure/populationCriteria/groups/QICore/QICoreMeasureGroups.tsx

```diff
@@ -907,7 +921,26 @@ const MeasureGroups = (props: MeasureGroupProps) => {
                     canEdit={canEdit}
                     placeholder="UCUM Code or Name"
                   />
-                </div>
+                  <TextField
```

Comment:
Do you think we could use https://github.com/MeasureAuthoringTool/madie-design-system/tree/main/react/components/NumberInput ? 
I know it is not mentioned in the story, but wondering if the input can only be a positive integer then may be we can restrict user and use this NumberInput, which basically uses TextField.

Reply:
Also this changes the way the TextField is displayed so this may be a question to Michael as well.

Reply:
Checked with Michael and he was good with the NumberInput change, so I updated